test/e2e/job.go

```diff
@@ -339,3 +332,26 @@ func newBool(val bool) *bool {
 	*p = val
 	return p
 }
+
+func TestJobsSetup(behavior, name string, rPol api.RestartPolicy, parallelism, completions int32) *batch.Job {
+	return newTestJob(behavior, name, rPol, parallelism, completions)
+}
+
+func TestJobsValidate(f *framework.Framework, job *batch.Job, completions int32) {
+	By("Creating a job")
```

Comment:
ok will change accordingly. my concern was that if we create job in setup. the job might be scheduled and finished completion even before we start upgrade process.